docs/pages/blog/mui-x-sep-2024-price-update.md

```diff
@@ -2,57 +2,75 @@
 title: Upcoming changes to MUI X pricing in 2024
 description: Check out the new pricing updates and how to transition to the new model.
 date: 2024-08-01T08:00:00.000Z
-authors: ['josefreitas']
+authors: ['josefreitas', 'oliviertassinari']
 tags: ['MUI X', 'Product']
 manualCard: true
 ---
 
-We're excited to share some important updates to our product offerings and pricing structure. This change reflects the expanded feature set and helps us continue delivering high-quality components to you and your users.
+Today, we are sharing some important updates to the MUI X product offerings and pricing structure.
+These changes are made to reflect the expanded feature set and help support the delivering of high-quality components to you and your users.
 
 ## Changes to the Pro plan
 
-### Current Pricing
+### Current pricing
 
-Under our current pricing model, if you licensed the Pro plan for 50 developers, you only paid for 10 seats. This concept was introduced when we offered only the Data Grid Pro with a much smaller feature set.
+Under the current pricing model, you only purchase a license for the first 10 developer.
+So to license the Pro plan for 50 developers you only need to purchase 10 licenses.
+This concept was introduced in the very beginning of the MUI X offering (2021) when we offered only the Data Grid Pro with a small feature set.
 
-### New Pricing
+Since the release of this component, we launched several new components and features, including [Date and Time Range Pickers](/x/react-date-pickers/date-time-range-picker/).
 
-With the launch of several new components, including [Date and Time Range Pickers](/x/react-date-pickers/date-time-range-picker/), as well as upcoming Pro versions of [Charts](/x/react-charts/) and [TreeView](/x/react-tree-view/), we're updating our pricing.
+### New pricing
 
-Moving forward, each developer requires a seat. So, for example, 50 developers will now need 50 seats.
+We will soon release the first Pro features of the [Charts](/x/react-charts/) and [Tree View](/x/react-tree-view/).
+These new features will only be available under the updated pricing: moving forward, each developer requires a license. So, for example, 50 developers will need 50 licenses.
 
-We're happy to discuss volume discounts to accommodate larger teams.
-
-### Grandfathering Current Customers
+### Grandfathering current customers
 
 We value our loyal customers and want to ensure a smooth transition: current customers will be grandfathered into their existing plans.
 
-This means you can continue using and renewing the legacy license for as long as you wish.
-However, the legacy license covers only Data Grid and Date Pickers.
+This means you can continue to renew your licenses under the legacy pricing for as long as you wish.
+However, the legacy pricing only covers the previous feature scope: Data Grid and Date Pickers.
```

Comment:
Rephrasing from "features scope" to "components" because I've seen users confused if there are differences in legacy Pro vs new Pro license key in Data Grid and Date pickers features as well.

```suggestion
However, the legacy pricing only covers the previously released components: Data Grid and Date Pickers.
```